src/http/client.cc

```diff
@@ -312,6 +312,15 @@ auto client::with_connection(Fn&& fn) {
     });
 }
 
+struct transport_error : public std::exception {
+    unsigned requests_submitted;
+    std::system_error lower;
+    transport_error(unsigned cs, const std::system_error& err) noexcept : requests_submitted(cs), lower(err) {}
```

Comment:
i am curious what "cs" is short for?

Reply:
:facepalm: it must be "reCwuests Submitted"

Reply:
Renamed :)